packages/concerto-core/lib/serializer/instancegenerator.js

```diff
@@ -190,7 +194,7 @@ class InstanceGenerator {
      * @throws {Error} if no concrete subclasses exist.
      */
     findConcreteSubclass(declaration) {
-        if (!declaration.isAbstract()) {
+        if (!declaration.isAbstract?.()) {
```

Comment:
Done. I'd prefer a terse check over an explicit check per declaration type, and given we have Union and Set declarations on the roadmap, we should probably add `isAbstract()` to those classes and MapDeclaration in order to avoid bloated conditional here.